lib/dom-parser.js

```diff
@@ -13,15 +13,15 @@ var XMLReader = sax.XMLReader;
 /**
  * Normalizes line ending according to https://www.w3.org/TR/xml11/#sec-line-ends:
  *
- * XML parsed entities are often stored in computer files which, 
- * for editing convenience, are organized into lines. 
- * These lines are typically separated by some combination 
+ * XML parsed entities are often stored in computer files which,
+ * for editing convenience, are organized into lines.
+ * These lines are typically separated by some combination
  * of the characters CARRIAGE RETURN (#xD) and LINE FEED (#xA).
```

Comment:
Shouldn't these lines which are quoted start with `>`, just like we did for `Well-formed constraint: No < in Attribute Values` in `lib/dom.js`?